### ISSUE TITLE
Image Post Format without Featured Image doesn't give any special treatment to the post image.

### ISSUE THREAD

**Issue by jeherve** [WordPress/twentysixteen#57]
Steps to get started:
1. Create a new post and use the Image Post Format.
2. Insert an image into the post.
3. Do not define a Featured Image.

In parallel, do the following:
1. Create a new post and use the Standard Post Format.
2. Define a Featured Image.

When comparing the 2 posts, I find that the standard post (with its Featured Image) actually does a better job of showing off my image.

I wonder if we should handle that special case:
- When a post uses the "Image" post format but does not include a featured image, improve `twentysixteen_post_thumbnail` to look for the first image in the post and display it instead of the featured image.
- Remove that first image from the post content.

What do you think?

**Comment by SeanChDavis** [comment]
A discussion is underway about what to do with post formats in the core theme, if anything at all. You can see it here: https://github.com/WordPress/twentysixteen/issues/45

I think the results you found are just based on how the theme treats Featured Images, which is to let them be full-width in the content column above the post title. Then the image in the content itself displays as you would expect in the post content. So basically nothing special was done at all for the Image post format.

I'd imagine any other approach requires special treatment based on post format and that opens the door to adjustments to all the other post formats as well. So it should probably be discussed at that level and if changes are going to happen, they trickle down from there.

That's my take on it, at least!

**Comment by iamtakashi** [comment]
At the moment, there is no plan to add special styling to image post format. I'm closing this to concentrate the discussion at #45.

**Comment by nishanttotla** [comment]
I'd like to open a new issue, but not sure if it has been discussed before. Whenever I create a standard post that contains images, the first image is picked by default as a featured image. Even when I remove featured image from post editing, it comes back. Has this problem been seen before?

Thanks,

UPDATE: Apologies, it was a problem with some custom code on my site, the theme works fine. I was concerned that the has_post_thumbnail() function was acting weird as has been reported earlier by some people, but that is not the case.
